libraries/core-react/src/components/Dialog/Dialog.tokens.ts

```diff
@@ -20,24 +19,41 @@ const {
   },
 } = tokens
 
-type Dialog = {
-  width: string
-  minHeight: string
-  background: string
-  borderRadius: string
-  spacingsMedium: string
-  title: Typography
-  description: Typography
-  boxShadow: string
-}
+type DialogToken = ComponentToken
 
-export const dialog: Dialog = {
+export const dialog: DialogToken = {
   width: '252px',
-  minHeight: '165px',
   background,
-  borderRadius,
-  spacingsMedium: spacingMedium,
-  title: accordion_header,
-  description: body_long,
-  boxShadow,
+  typography: accordion_header,
+  border: {
+    type: 'border',
+    radius: borderRadius,
+  },
+  spacings: {
+    top: spacingMedium,
+  },
+  entities: {
+    children: {
+      spacings: {
+        top: '0px',
+        bottom: spacingMedium,
+        left: spacingMedium,
+        right: spacingMedium,
+      },
+    },
+    title: {
+      minHeight: '24px',
+      height: '8px',
+      typography: accordion_header,
+    },
+    content: {
+      minHeight: '80px',
+      height: '104px',
+      typography: body_long,
+    },
+    actions: {
+      minHeight: '48px',
+      height: '8px',
```

Comment:
This just looks weird, `minHeight` is `48px` and `height` is `8px`? 😂

Do we need to do trickery when no children i present? Can't it just be `0` in the code? Not sure this is a value worth having in token as its very niche specific.

Reply:
Yes I agree 🤣  I'll have a look. But if we still need it I can move `height: '8px'` to `children` where it's more fitting I think

Reply:
`!children` ..Or not 🤔

Reply:
I found out that the `height` was to avoid grid filling up space whenever a user forgot to add content inside `Title` or `Actions`